FirebaseCore.podspec

```diff
@@ -53,7 +53,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   # Remember to also update version in `cmake/external/GoogleUtilities.cmake`
   s.dependency 'GoogleUtilities/Environment', '~> 8.0'
   s.dependency 'GoogleUtilities/Logger', '~> 8.0'
-  s.dependency 'FirebaseCoreInternal', '~> 11.4'
+  s.dependency 'FirebaseCoreInternal', '>= 11.4.2', '< 12.0'
```

Comment:
Since we are moving towards https://github.com/firebase/firebase-ios-sdk/pull/13953, I'm thinking this range should be tighter?